templates/project/.symfony.bundle.yaml.twig

```diff
@@ -0,0 +1,47 @@
+# DO NOT EDIT THIS FILE!
+#
+# It's auto-generated by sonata-project/dev-kit package.
+
+# MANDATORY
+# An array with the name of the branches for which you want to publish docs
+# It doesn't have to be an exhaustive list of your repository branches, but
+# only the main repository branches (past or present)
```

Comment:
Should we make the comments Twig comments instead of YAML comments? That way they will not be copied in every repository, I think they are only useful when you edit the file, and having them here contradicts the first comment about how this file should not be edited directly on repositories.

Reply:
I like the idea